Mining Vendor Redo

katnep opened the pull request:
basically Re adds all the stuff i added back at the start of 6th port,
also re added stimpacks to the lsit becuase i don't see why they needed
to be removed from it

katnep committed:
Mining Vendor Redo

basically Re adds all the stuff i added back at the start of 6th port,
also re added stimpacks to the lsit becuase i don't see why they needed
to be removed from it

TalkingCactus commented on code/modules/mining/machine_vending.dm:
```diff
@@ -51,6 +51,11 @@
 		new /datum/data/mining_equipment("Drone AI Upgrade",	/obj/item/slimepotion/sentience/mining,									1000),
 		new /datum/data/mining_equipment("Jump Boots",			/obj/item/clothing/shoes/bhop,											2500),
+		new /datum/data/mining_equipment("Survival Knife",		/obj/item/weapon/kitchen/knife/combat/survival,							350),
+		new /datum/data/mining_equipment("Syringe Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2250),
```
This needs a more descriptive name.

katnep replied:
done changed to 'survival pen economy pack'

TalkingCactus commented on code/modules/mining/machine_vending.dm:
```diff
@@ -51,6 +51,11 @@
+		new /datum/data/mining_equipment("Syringe Economy Pack",/obj/item/weapon/storage/box/medipens/survival,							2250),
+		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,			250),
+		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1250),
+		new /datum/data/mining_equipment("Whetstone",			/obj/item/weapon/sharpener,												400),
```
Whetstones can stay gone.

Roland410 replied:
Whetstones aren't a REALLY bad thing and I don't understand why they were removed anyways.

katnep replied:
removed

TalkingCactus commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -179,11 +179,23 @@
 	desc = "A box with several stimpack medipens for the economical miner."
 	illustration = "syringe"
 
+
```
Excessive line spacing, remove one.

katnep replied:
removed the line

TalkingCactus replied:
The line is still there.

katnep committed:
whetstone removed

whetstone removed

katnep committed:
price balances

just tweaked the prices of the new bundles to make them actually worth
purchasing following a 5 for the price of 4 standard

katnep committed:
The final change

finishes the requested changes

TalkingCactus commented on code/modules/mining/machine_vending.dm:
```diff
@@ -52,10 +52,9 @@
 		new /datum/data/mining_equipment("Stimpack",			/obj/item/weapon/reagent_containers/hypospray/medipen/stimpack,			250),
-		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1250),
-		new /datum/data/mining_equipment("Whetstone",			/obj/item/weapon/sharpener,												400),
+		new /datum/data/mining_equipment("Stimpack Economy Pack",/obj/item/weapon/storage/box/medipens/utility,							1000),
```
Trailing comma, remove it.

TalkingCactus committed:
removes excessive line spacing

TalkingCactus committed:
removes trailing comma
